src_erl/Communication_Layer/http_Nerlserver/src/newparser.erl

```diff
@@ -0,0 +1,121 @@
+%%%-------------------------------------------------------------------
+%%% @author kapelnik
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 19. Apr 2021 4:27 AM
+%%%-------------------------------------------------------------------
+-module(newparser).
```

Comment:
newparser is a bad name for a module. 
You can call it binaryParser or something with meaning.